CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+_9.5.0_
+- **[ENHANCEMENTS]**
+  - Allow extension capability keys to contain dot characters  [#2271](https://github.com/appium/java-client/pull/2271)
```

Comment:
as part of https://github.com/appium/java-client/pull/2304/files#diff-6b8067d4d88e05a0de63fec770cc02ea014ff9d0f4384f498055d72c04d4f666R25 I've also added two new option classes for prebuiltWDAPath and usePreinstalledWDA xcuitest capabilities